packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -493,6 +493,41 @@ class ShaderMask extends SingleChildRenderObjectWidget {
 /// [ImageFiltered] instead. For that scenario, [ImageFiltered] is both
 /// easier to use and less expensive than [BackdropFilter].
 ///
+/// {@tool snippet}
+///
+/// This example shows how the common case of applying a [BackdropFilter] blur
+/// to a single child can be replaced with an [ImageFiltered] widget. This code
+/// is generally simpler and the performance will be improved dramatically.
+///
+/// The implementation below is unnecessarily expensive.
+///
+/// ```dart
+///  Stack(
+///    children: [
+///      Positioned.fill(child: Image.asset('image.png')),
+///        Positioned.fill(
```

Comment:
Bad indentation? Shouldn't this line up with the previous Positioned?

Reply:
Done